client/src/components/relatedItems/RelatedItemCard.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { RiInformationLine } from 'react-icons/ri';
 import PropTypes from 'prop-types';
-import rounder from '../../../../rounder';
+import helper from '../../../../helper';
```

Comment:
Well done with that helper.js, took that to the next level! Thanks for putting it in my file already LEGEND!